spec/unit/outputs/opensearch_spec.rb

```diff
@@ -58,7 +62,12 @@
         "index" => "my-index",
         "hosts" => ["localhost","localhost:9202"],
         "path" => "some-path",
-        "manage_template" => false
+        "manage_template" => false,
+        "auth_type" => {
```

Comment:
May be you should all test cases for aws_iam as new test cases?

Reply:
yes, will add later